File: python/armstrong-numbers/armstrong_numbers.py
def is_armstrong(number):
    pow = len(str(number))
    sum = 0
    temp = number
    while temp > 0:
        reminder = temp % 10
        # raise by pow
        sum += reminder ** pow
        # floor by 10
        temp //=10
    if number == sum :
        return True
    else:
        return False

File: typescript/armstrong-numbers/armstrong-numbers.ts
export function isArmstrongNumber(number: number): boolean {
  let pow = number.toString().length;
  let sum = 0;
  let temp = number;
  while (temp > 0) {
    let reminder = temp % 10;
    // raise by pow
    sum += reminder ** pow;
    // floor by 10
    temp = ~~(temp / 10);
  }
  if (number === sum) {
    return true;
  } else {
    return false;
  }
}
